src/commands/dev.ts

```diff
@@ -148,6 +149,17 @@ export async function command(commandOptions: CommandOptions) {
     }
   }
 
+  function addToDependents(spec: string, fileLoc: string) {
+    if (!spec.includes('web_modules') && !spec.includes('node_modules') && !fileLoc.includes('web_modules') && !fileLoc.includes('node_modules')) {
+      const dependents = dependentsTree.get(spec);
+      if (dependents) {
+        dependentsTree.set(spec, dependents.add(fileLoc));
```

Comment:
I don't think that you need to dependentsTree.set here, since it already has the reference to `dependents`